server/mediaserver.go

```diff
@@ -290,14 +290,23 @@ func getHLSMediaPlaylistHandler(s *LivepeerServer) func(url *url.URL) (*m3u8.Med
 			return nil, ErrNotFound
 		}
 
+		//If the hls stream is not empty, we don't need to request from the network anymore.  This could happen if there is a local broadcaster.
+		pl, err := hlsStrm.GetVariantPlaylist(strmID.String())
+		if err != nil {
+			glog.Errorf("Error getting pl")
+		}
+		if pl.Segments[0].URI != "" {
+			return pl, nil
+		}
+
 		//Subscribe from the network if we can't find a subscriber
 		sub, err := s.LivepeerNode.VideoNetwork.GetSubscriber(strmID.String())
 		if err != nil {
 			glog.Errorf("Error getting subscriber for %v: %v", strmID, err)
 			return nil, err
 		}
 		if !sub.IsWorking() {
-			glog.Infof("Making new subscriber for %v", strmID)
+			glog.V(4).Infof("Making new subscriber for %v", strmID)
```

Comment:
Why is this the only log statement where a level is set?

Reply:
Oh... I was working on the log cleanup in another branch, but haven't merged yet.  But the code has changed a lot, so I may just do the log cleanup later.